### PR TITLE
Dictionary for log messages

### PR DESCRIPTION
**Pull request by Christian-B**
This a a multiple repository PR most must be done at the same time.

Documentation is at:
http://spinnakermanchester.github.io/spynnaker/5.0.0/Makefiles.html

Do not merge until this message is removed.

---
Main change here is rename of all common.mk and Makefile.common files found in multiple places!

### CODE REVIEW

**Review by dkfellows — COMMENTED** [review]
I approve of the file renaming.

File: release.txt [hunk 1]
```diff
@@ -4,6 +4,10 @@
 #
 #-------------------------------------------------------------------------------
 
+Version 5.0.0
```
Comment (rowleya) [inline release.txt]: This probably makes this version 4.0.0 - API incompatible (since the makefile names have changed), but no need to double-jump

Comment (rowleya) [reply release.txt]: Actually, this isn't API incompatible as the old makefile imports the new... maybe 3.2.0?

Comment (lplana) [reply release.txt]: 3.2.0 is already out in the open.

I don't know if this is a new feature (3.3.0) or a new implementation of the same thing (3.2.1).

Comment (rowleya) [reply release.txt]: Yes, maybe 3.2.1 really :)

Comment (lplana) [reply release.txt]: I just checked release.txt on master and there is already a '3.2.1'.

Comment (rowleya) [reply release.txt]: Haha!  I guess 3.2.2!

**Comment by rowleya** [comment]
Tested in isolation and it works fine

**Pull request merged**